src/main/java/spoon/reflect/factory/Factory.java

```diff
@@ -787,6 +795,11 @@
 	 */
 	CtQuery createQuery(Object input);
 
+	/**
+	 * @see QueryFactory#createQuery(Object...)
+	 */
+	CtQuery createQuery(Object[] input);
```

Comment:
would that `createQuery(Object... input)` work?

Reply:
Actually it should not be there, as I created those methods in #1739  I'll remove it.